src/components/layouts/Layout.js

```diff
@@ -76,17 +76,21 @@ const Layout = ({ children, initialContext, hasSideBar, location }) => {
               Skip to Main Content
             </SprkLink>
           </div>
-          <div
-            className="sprk-u-BackgroundColor--black
-            sprk-u-pvm
-            sprk-u-AbsoluteCenter"
-          >
-            <a
-              href="https://v13--spark-design-system.netlify.app/"
-              className="docs-c-Banner--link sprk-u-mlm"
-            >
-              View the previous major release of Spark
-            </a>
+          <div className="sprk-u-BackgroundColor--black sprk-u-AbsoluteCenter sprk-u-pam">
+            <p className="sprk-u-Color--white">
+              <span className="sprk-u-FontWeight--bold sprk-u-mrs">
+                Designs launching before July 14, 2021
+              </span>
+              should reference
+              <a
+                href="https://sparkdesignsystem.com/"
```

Comment:
Hey @diriquelme  this is the wrong link. It takes them back to where they are. You want the v13 link here.

Reply:
Link has been updated to point to v13 https://v13--spark-design-system.netlify.app/